recipes/fmi4cpp/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} -GNinja .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_SHARED_LIBS:BOOL=ON \
+      -DBUILD_TESTING:BOOL=ON ..
+
+cmake --build . --config Release
+cmake --build . --config Release --target install
+ctest --output-on-failure -C Release
```

Comment:
Logs show that no tests are being run. Is that what you want? Existence tests are sufficient for conda-forge.

Reply:
Thanks for noticing, whenever I can I also enable "regular" tests so that they can detect if there is something unexpected in the conda-forge-provided dependencies, but for sure it is not useful if not test runs.

Reply:
Apparently fmi4cpp is using a custom CMake option to enable tests, I added it in https://github.com/conda-forge/staged-recipes/pull/18411/commits/29151a7f4d9aab0403e9b87629f16f41c85fc4fc and https://github.com/conda-forge/staged-recipes/pull/18411/commits/04781726527992129f0818672691b84598716b87 so that tests are actually running.

Reply:
Apparently the 0.7.0 release of conan does not have tests. However, tests are present in the master version (see https://github.com/NTNU-IHB/FMI4cpp/blob/31cd95965da94652597e340c4b1b78c93fb830ec/CMakeLists.txt#L90) so if for you it is ok I would leave this here so that tests are going to run as soon as they are available.

Reply:
Leaving this is fine since the intention is to have tests soon.